tap_tiktok_ads/sync.py

```diff
@@ -19,10 +19,14 @@ def update_currently_syncing(state, stream_name):
     singer.write_state(state)
 
 def sync(tik_tok_client, config, state, catalog):
+
+    sandbox_mode = False if config.get("sandbox",False) in  ("false","FALSE",False) else True
```

Comment:
So instead of re-evaluating the sandbox value i have made it accessible through tiktokclient class.
on the web app this also uses a checkbox, so other values like "false","FALSE",False wont need evaluation.